src/coreclr/tests/src/profiler/native/classfactory.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
```

Comment:
We should use the repo standard license header: https://github.com/dotnet/runtime/blob/master/CONTRIBUTING.md#file-headers